deferred-iterator.js

```diff
@@ -27,6 +27,10 @@ DeferredIterator.prototype._operation = function (method, args) {
   DeferredIterator.prototype['_' + m] = function () {
     this._operation(m, arguments)
   }
+  // must defer the full call, else error 'this.db._serializeKey is not a function'
+  DeferredIterator.prototype['seek'] = function () {
```

Comment:
Does this need to be inside the forEach loop?

Reply:
oops.. :P - done

Reply:
the alternative would be to defer _serialiseKey and _serialiseValue

Reply:
Right. That might have unintended consequences though, so let's go with deferring the public `seek()` for now.